calc/HamiltonianTwo.cpp

```diff
@@ -551,10 +551,10 @@ void HamiltonianTwo::calculate(const Configuration &conf_tot) {
 
     std::cout << ">>TOT" << std::setw(7) << nSteps_two*symmetries.size() << std::endl;
 
-#pragma omp parallel for collapse(2) schedule(static, 1)
+#pragma omp parallel for schedule(static, 1)
```

Comment:
Same here. Concerning static scheduling, the ordering is requested to build the potential lines starting from one site of the plot window and ending up on the other site. This successive way of plotting allows to start the drawing of potential lines already before everything is calculated. In my opinion, your suggestion won't work here. We really need static parallelization.